conftest.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+from fake_zyte_api.main import make_app
+
+if TYPE_CHECKING:
```

Comment:
Just curious, why is it needed?

Reply:
If you mean `if TYPE_CHECKING` then it's usually done to simplify the imports done when running the code, to help with both performance and cyclic imports.

Reply:
Aha, I see - pytest uses argument names for dependency injection, not argument types.